src/syntax/FStar.Syntax.Util.fst

```diff
@@ -1233,8 +1233,9 @@ let attr_eq a a' =
    | Equal -> true
    | _ -> false
 
+let attr_substitute_lid = lid_of_path ["FStar"; "Pervasives"; "Substitute"] Range.dummyRange
```

Comment:
Sorry to nitpick, but this should go in `FStar.Parser.Const`

Reply:
No no, thanks for the feedback, just rebased that!